README.md

```diff
@@ -92,16 +112,23 @@ At runtime, the value of `My.Assets.star` will be something like
   module.exports = {
     ...
     elmAssetsLoader: {
-      localPath: function(path) {
-        return path.replace(/^\/public\//, "")
+      localPath: function(url) {
+        // transform `url` to a local path that resolves to a file
+        return url.replace(/^\/public\//, "")
```

Comment:
Does this ever appear like `public/` without the leading slash?

Put another way, would this be better or worse as `/^\/?public\//` instead?